src/Provider/Provider.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -100,7 +100,7 @@ private List<VacancySummary> GetFilteredVacancies(List<VacancySummary> vacancies
                     break;
                 case FilteringOptions.ClosingSoonWithNoApplications:
                     filteredVacancies = vacancies.Where(v =>
-                        v.ClosingDate <= _timeProvider.Today.AddDays(5) && v.Status == VacancyStatus.Live && (v.NoOfSuccessfulApplications == 0 || v.NoOfUnsuccessfulApplications == 0 ||
+                        v.ClosingDate <= _timeProvider.Today.AddDays(5) && v.Status == VacancyStatus.Live && (v.NoOfSuccessfulApplications == 0 && v.NoOfUnsuccessfulApplications == 0 &&
                                                                      v.NoOfNewApplications == 0));
```

Comment:
Would assume NoOfNewApplications is already in the other two properties being evaluated there is excessive.